.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Publish Python 🐍 distribution 📦 to TestPyPI
+
+on:
+  workflow_dispatch:  # Keep manual trigger
+    inputs:
+      rc_version:
+        description: 'RC version number (e.g., 1, 2, 3)'
+        required: true
+        type: string
+  schedule:
+    - cron: "0 0 * * *"  # Run every day at midnight
+
+jobs:
+  trigger-client-and-models-build:
+    name: Trigger llama-stack-client and llama-models build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get date
+      id: date
+      run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
+    - name: Compute version based on dispatch event
+      id: version
+      run: |
+        # Read base version from pyproject.toml
+        if [ "${{ github.event_name }}" = "schedule" ]; then
+          echo "version=.dev${{ steps.date.outputs.date }}" >> $GITHUB_OUTPUT
+        elif [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+          echo "version=rc${{ inputs.rc_version }}" >> $GITHUB_OUTPUT
+        else
+          echo "version=.dev$(shuf -i 10000000-99999999 -n 1)" >> $GITHUB_OUTPUT
```

Comment:
This would overlap with the range of date (when triggered on schedule) and potentially conflict?

Reply:
I don't think there will be a conflict as this is a 8 digit number, and date is a 6 digit number. This is there for testing the workflow on push, we won't reach here other than testing, since we will either manually trigger or have nightly schedules.

Reply:
I should not have counted the number of digits late at night :-) You are right. This should be fine.